### DISCUSSION TITLE
404 Invite

### DISCUSSION THREAD

**Discussion by Wasserkleber** [community/community#56506] [New to GitHub]
I got Invited by a Friend to a repository, but when I try to join I get a 404. But why?

**Comment by mrepol742** [comment]
Check if you are signed in with Github or if the invitation is already expired.

**Reply by Wasserkleber** [reply]
How can I see if its expired?

**Reply by mrepol742** [reply]
Itll expired in 7 days. Try contacting your friend to reinvite you or maybe he invited a wrong person.
